app/src/main/java/net/kwatts/powtools/database/RideDao.java

```diff
@@ -0,0 +1,31 @@
+package net.kwatts.powtools.database;
+
+import android.arch.persistence.room.Dao;
+import android.arch.persistence.room.Insert;
+import android.arch.persistence.room.OnConflictStrategy;
+import android.arch.persistence.room.Query;
+import java.util.List;
+
+@Dao
+public interface RideDao {
+    @Query("SELECT * FROM Ride")
```

Comment:
Having mainly using query builders & ContentProvider for SQlite, I was - at first glance - surprised to see the raw SQLite there. I then looked into Room and think I get what it's going after. It seems fine, just might make it tricky to expose DB data to other components in the future that need a cursor - like SyncAdapter, if we want to persist this on a server. I'm sure it's not a big deal, probably can just return a Cursor from the DAO object.

Reply:
would be very surprising if we can't get a cursor. did you look into that?

Reply:
yep DAO returns Cursor, 
@Query("SELECT * FROM Ride")
Cursor getCursorAll();